altos-rust/altos-core/libs/heap/free_list_allocator/src/test.rs

```diff
@@ -131,9 +131,9 @@ impl TestFreeList {
 
     // Helper function to check that every block in the list satisfies some condition
     // Returns false if the condition returns false for any node
-    pub fn each_free_block_satisfies<F: Fn(&free_list::BlockHeader) -> bool>
-        (&self, condition: F) -> bool {
-
+    pub fn each_free_block_satisfies<F>(&self, condition: F) -> bool
+        where F: Fn(&free_list::BlockHeader) -> bool
+    {
```

Comment:
Having the curly bracket on the next line isn't the way we're doing it in other places, like `free_list.rs`, so this seems like inconsistent style.